habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1084,6 +1084,37 @@ def step(self, task: EmbodiedTask, *args: Any, **kwargs: Any):
         return self._sim.get_observations_at()  # type: ignore
 
 
+@registry.register_task_action
```

Comment:
File `habitat-lab/habitat/tasks/nav/nav.py` is already overloaded and too long. Let's move out all continuous action related declaration into separate file something like `continuous_nav.py` or `velocity_nav.py.`